docs/standard-for-public-code.md

```diff
@@ -0,0 +1,201 @@
+# The Standard for Public Code's compliance to the criteria and requirements of the Standard for Public Code version 0.3.0
+
+The Standard for Public Code is not a codebase created in the context of a policy mandate, thus some criteria do not apply directly.
+
+Link to commitment to meet the Standard for Public Code:
```

Comment:
Should this be a hyperlink?
>Link to commitment to meet the Standard for Public Code:

Reply:
No, but we should add a link to the CONTRIBUTING file where we state that contributions should adhere to it.